src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
```

Comment:
```suggestion
  &:not(:last-child) {
```
and move into &__star

Reply:
If I remove this space but the function does not work for me